mne/event.py

```diff
@@ -299,8 +298,9 @@ def read_events(filename, include=None, exclude=None, mask=None,
     return out
 
 
+@fill_doc
 def write_events(filename, event_list):
```

Comment:
For now I changed the docstring to %(events)s; but here the events array is actually called event_list..
How much pain is it to change the argument name from event_list to events for consistency?